packages/project-management-automation/lib/add-milestone.js

```diff
@@ -34,6 +33,37 @@ const isDuplicateValidationError = ( error ) =>
 	Array.isArray( error.errors ) &&
 	error.errors.some( ( { code } ) => code === 'already_exists' );
 
+/**
+ * Returns a promise resolving to a milestone by a given title, if exists.
+ *
+ * @param {GitHub} octokit Initialized Octokit REST client.
+ * @param {string} owner   Repository owner.
+ * @param {string} repo    Repository name.
+ * @param {string} title   Milestone title.
+ *
+ * @return {Promise<OktokitIssuesListMilestonesForRepoResponseItem|void>} Promise resolving to milestone, if exists.
+ */
+async function getMilestoneByTitle( octokit, owner, repo, title ) {
+	const options = octokit.issues.listMilestonesForRepo.endpoint.merge( {
+		owner,
+		repo,
+	} );
```

Comment:
I think there's an issue here: We now rely on fetching the _previous_ milestone, which is most likely going to be in a closed state. By default, the milestones listing endpoint will only include open milestones.

https://developer.github.com/v3/issues/milestones/

Might also need to change default `direction`, since `due_on` `asc` is probably not what we want.

Reply:
Should be fixed now by c00140f.